pkg/operator/starter.go

```diff
@@ -139,14 +138,30 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 		return err
 	}
 
+	volumeSnapshotClassAsset, err := assets.ReadFile("volumesnapshotclass.yaml")
+	if err != nil {
+		return err
+	}
+	volumeSnapshotCtrl := volumesnapshotclasscontroller.NewVolumeSnapshotClassController(
+		"VolumeSnapshotController",
+		volumeSnapshotClassAsset,
+		infraInformer,
+		dynamicClient,
+		operatorClient,
+		time.Minute,
```

Comment:
nit: I was wondering if we could increase this to something like 10 minutes, since the resource ID shouldn't change. We have other controllers with this tight sync interval as well.

Reply:
I have 1 minute for users changing the VolumeSnapshotClass. Otherwise it could be 1 hour or more.